arXiv.org.js

```diff
@@ -109,6 +115,10 @@ function doWeb(doc, url) {
 	}
 	else {
 		var id;
+		var versionMatch = url.match(/v(\d+)(\.pdf)?$/);
```

Comment:
This breaks for anchor & social media links (like https://arxiv.org/abs/1805.09170v1#abs ) Could we do something like
`url.replace(/[?#].+$/, "").match...` ? 
Otherwise looks good, so please go ahead & merge

Reply:
Updated in the new version